plugins/modules/zos_backup_restore.py

```diff
@@ -187,6 +187,7 @@
     description:
       - Specifies the new HLQ to use for the data sets being restored.
       - Defaults to running user's username.
+      - Use keyword N to restores datasets to their original HLQ from when they were archived.
```

Comment:
Discussed this change with Marcel and we agreed that using the same option already available to users but reserving a special keyword (or key letter) would be more straightforward for everybody instead of adding a new option to the module's interface. Probably better to use 'K' (as in KEEP) or 'R' (as in RESTORE) though. Should not conflict with valid HLQs since it is just one character.